### PR TITLE
Build multiplatform images when issue `make release-images`

### LINKED ISSUE #97458: 'make release-images' doesn't build ppc64le/s390x images
**What happened**:
When run below commands against master branch, only amd64 images are built.
```
export KUBE_BUILD_PLATFORMS="linux/amd64 linux/ppc64le linux/s390x"
export KUBE_BUILD_PULL_LATEST_IMAGES=y
make release-images
```
**What you expected to happen**:
Images should be built successfully according to values set in `KUBE_BUILD_PLATFORMS`.
**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration:
- OS (e.g: `cat /etc/os-release`):
- Kernel (e.g. `uname -a`):
- Install tools:
- Network plugin and version (if this is a network-related bug):
- Others:

### PR DESCRIPTION
**Pull request by yu2003w**
Signed-off-by: jared <yu2003w@hotmail.com>

**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #97458

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
None
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by fejta** [comment]
I am not convinced we should always use this pattern

**Comment by yu2003w** [comment]
Hi @fejta ,
Do you think the change is not necessary. 
Or do I only need to run `DOCKER_CLI_EXPERIMENTAL=enabled docker buildx build` for non-x86 platforms?
I found that pause image also use the same pattern.
Thanks for your comments.

**Comment by fejta** [comment]
/ok-to-test

I am primarily concerned about the effect of changing from `docker build` to `docker buildx` and would ideally like to maintain the current way of building: at least by default, at least for amd64.

AKA I would like people to be able to opt-into and then opt-out of this change and not just sending it to everyone immediately.

Alternatively I guess you can try and explain me why I shouldn't worry about this change on amd64 :)

**Comment by fejta** [comment]
Also are you familiar with `make cross`?

**Comment by yu2003w** [comment]
Hi @fejta ,
Per my understanding, `make cross` only compiled binaries for different platforms.
This fix is to build images for different platforms. 
I updated PR to run `docker buildx buid` for `linux/ppc64le` and `linux/s390x`. This change will not impact other platforms.

**Review by fejta — APPROVED** [review]
/lgtm
/approve

File: build/lib/release.sh [hunk 1]
```diff
@@ -383,7 +383,11 @@ EOF
           echo "COPY nsswitch.conf /etc/" >> "${docker_file_path}"
         fi
 
-        "${DOCKER[@]}" build ${docker_build_opts:+"${docker_build_opts}"} -q -t "${docker_image_tag}" "${docker_build_path}" >/dev/null
+	if [[ "${arch}" == "linux/s390x" || "${arch}" == "linux/ppc64le" ]]; then
+          DOCKER_CLI_EXPERIMENTAL=enabled "${DOCKER[@]}" buildx build --platform linux/"${arch}" --load ${docker_build_opts:+"${docker_build_opts}"} -q -t "${docker_image_tag}" "${docker_build_path}" >/dev/null
```
Comment (BenTheElder) [inline build/lib/release.sh:387]: why aren't we using buildx on all paltforms?
cc @dims @justaugustus @vinayakankugoyal

Comment (BenTheElder) [reply build/lib/release.sh:387]: it seems https://github.com/kubernetes/kubernetes/issues/97458 was also only solved for the specific list of platforms in the issue?

**Comment by BenTheElder** [comment]
`make cross` builds for _all_ platforms.

Currently if you want a multi-arch image for less platforms, you build for the N platforms (in separate invocations) and assemble your own manifest image. That should work.

It makes sense to support building for a subset of platforms into a manifest image directly, but I don't think that should special-case two platforms.

**Pull request merged**